frontend/src/component/feature/FeatureView/FeatureOverview/FeatureOverviewEnvironments/FeatureOverviewEnvironment/EnvironmentAccordionBody/EnvironmentAccordionBody.tsx

```diff
@@ -195,21 +213,68 @@ const EnvironmentAccordionBody = ({
                 <ConditionallyRender
                     condition={strategies.length > 0}
                     show={
-                        <>
-                            {strategies.map((strategy, index) => (
-                                <StrategyDraggableItem
-                                    key={strategy.id}
-                                    strategy={strategy}
-                                    index={index}
-                                    environmentName={featureEnvironment.name}
-                                    otherEnvironments={otherEnvironments}
-                                    isDragging={dragItem?.id === strategy.id}
-                                    onDragStartRef={onDragStartRef}
-                                    onDragOver={onDragOver(strategy.id)}
-                                    onDragEnd={onDragEnd}
-                                />
-                            ))}
-                        </>
+                        <ConditionallyRender
+                            condition={
```

Comment:
We want to show the normal behavior when strategies are fewer than 50 and when the flag is disabled.